filebeat/input/journald/pkg/journalfield/matcher.go

```diff
@@ -50,8 +50,12 @@ type journal interface {
 	AddConjunction() error
 }
 
-var defaultBuilder = MatcherBuilder{Conversions: journaldEventFields}
-var kernelMatcher = Matcher{"_TRANSPORT=kernel"}
+var (
+	defaultBuilder = MatcherBuilder{Conversions: journaldEventFields}
+	coreDumpMsgID  = MustBuildMatcher("message_id=fc2e22bc6ee647b6b90729ab34a250b1")
+	journaldUID    = MustBuildMatcher("journald.uid=0")
+	journaldPID    = MustBuildMatcher("journald.pid=1")
```

Comment:
I wonder if we could somehow link to a documentation from journald showing those magic strings here. From your comment regarding the community journal beat I got to this function: https://github.com/systemd/systemd/blob/641e2124de6047e6010cd2925ea22fba29b25309/src/shared/logs-show.c#L1409-L1455

This is far from being a good documentation but it might be better than nothing. What do you think?